src/app/find-help/search-result/search-result-attribute/search-result-attribute.component.scss

```diff
@@ -11,3 +11,7 @@
   color: var(--c-yellow);
   margin-right: 0.5em;
 }
+
+.first-case-up::first-letter {
+  text-transform: uppercase;
```

Comment:
Trzeba być bardzo ostrożnym przy stylowaniu `::first-letter`, bo w Firefoksie rozmiar zawartości inline jest liczony na podstawie pierwszego znaku, więc jeśli np. byłoby tam jakieś tło, a tu zmienialibyśmy pierwszej literze rozmiar fonta, to by się wszystko rozjechało. 😄 Ale `uppercase` jest jeszcze OK.

Docelowo backend powinien to po prostu lepiej zwracać.

Reply:
To fakt, napisałem u nich na kanale.